app/models/miq_alert.rb

```diff
@@ -501,7 +501,7 @@ def self.expression_by_name(name)
 
   def self.raw_events
     @raw_events ||= expression_by_name("event_threshold")[:options].find { |h| h[:name] == :event_types }[:values] +
-                    ['hawkular_event']
+                    ['hawkular_alert']
```

Comment:
I think that @moolitayer was also interested on this.
This is just a renaming to only show desired events on ManageIQ.
I think this would not impact on the Datawarehouse provider, but just in case if you want to take a look as well.